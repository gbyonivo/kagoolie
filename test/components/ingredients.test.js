import React from 'react';
import renderer from 'react-test-renderer';
import Ingredients from '../../src/components/ingredients';
import store from '../__testData__/store';

describe('Ingredients component', () => {
  it('renders correctly', () => {
    const actual = renderer.create(<Ingredients ingredients={store.beersReducer.beers[0].ingredients}/>);
    expect(actual).toMatchSnapshot();
  });
});