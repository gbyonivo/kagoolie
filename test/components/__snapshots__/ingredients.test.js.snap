// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ingredients component renders correctly 1`] = `
<div
  className="ingredientsWrapper"
>
  <h6>
    <em>
      Ingredients
    </em>
  </h6>
  <div
    className="ingredients"
  >
    <div
      className="ingredient"
    >
      malt
    </div>
    <div
      className="ingredientDetails"
    >
      <span>
        malter
        <span
          className="ingredientAmount"
        >
           undefined
          undefined, 
        </span>
      </span>
    </div>
  </div>
  <div
    className="ingredients"
  >
    <div
      className="ingredient"
    >
      yeast
    </div>
    <div
      className="ingredientDetails"
    >
      salah
    </div>
  </div>
</div>
`;
